tetris.py

```diff
@@ -187,6 +188,17 @@ def new_game():
     tetris_control.game = game
 
 
+def start_dialogue():
+    if tetris_control.game.game_status == GameStatus.in_progress or tetris_control.game.game_status == GameStatus.paused:
+        tetris_control.pause_game()
+        if mb.askokcancel('End current game?', 'Do you want to end the current game and start anew?', parent=topbar_time):
+            new_game()
+        else:
+            tetris_control.pause_game()
```

Comment:
I don't think this logic works if the game is already paused when you click new game button.